scripts/run_e2e_tests.py

```diff
@@ -97,7 +97,7 @@
     help='If true, skips building files. The default value is false.',
     action='store_true')
 _PARSER.add_argument(
-    '--sharding-instances', type=int, default=3,
+    '--sharding-instances', type=int, default=1,
```

Comment:
Should we set this only for the tests which are failing with `unable to bind error`?

Reply:
done